src/components/MonthCalendar.spec.tsx

```diff
@@ -0,0 +1,133 @@
+import MonthCalendar from './MonthCalendar';
+import { mountWithProviders } from '../utils/testing';
+import { ZetkinEvent } from '../types/zetkin';
+
+describe('MonthCalendar', () => {
+    let dummyMonth : number;
+    let dummyYear: number;
+    let dummyEvents: ZetkinEvent[];
+    const dummyStartTime = '2021-05-10T13:37:00+00:00';
+    const dummyEndTime = '2021-05-10T14:37:00+00:00';
+
+    beforeEach(() => {
+        dummyMonth = 4;
+        dummyYear = 2021;
+        cy.fixture('dummyEvents.json')
+            .then((data: { data: ZetkinEvent[] }) => {
+                dummyEvents = data.data;
+                dummyEvents[0].start_time = dummyStartTime;
+                dummyEvents[0].end_time = dummyEndTime;
+                dummyEvents[1] = {
+                    ...dummyEvents[0],
+                    'end_time': '2021-05-10T17:37:00+00:00',
+                    'id': 25,
+                    'start_time': '2021-05-10T15:37:00+00:00',
+                };
+                dummyEvents[3] = {
+                    ...dummyEvents[0],
+                    'end_time': '2021-04-27T17:37:00+00:00',
+                    'id': 24,
+                    'start_time': '2021-04-27T15:37:00+00:00',
+                };
+            });
+    });
+
+    it('shows a grid with 6 rows and 7 columns', () => {
```

Comment:
I think this shouldn't be locked to 6 rows. Some months can fit on five rows (or even four rows for the odd February that starts on a Monday, like this year!). When the month fits, I see no reason not using the full vertical space available to show the selected month, which means each row can be taller, and fit more events without overflowing.